packages/phone-number-privacy/combiner/src/common/handlers.ts

```diff
@@ -86,9 +126,98 @@ export function meteringHandler<R extends OdisRequest>(
   }
 }
 
+export function timeoutHandler<R extends OdisRequest>(
```

Comment:
So the combiner didn't have any top level timeout logic like this before, and I'm hesitant to add it now. The timeoutMs value that is being used from the config is the same one that is used to timeout the requests to the signers. I'd be in favor of removing this just to decrease complexity and revisit whether another timeout is needed after the launch. What do you think?